content/notebooks/hydrology/03_Watershed_properties.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    grid.plot()
</pre><p>This is not working on the PAVICS beta image  <code>grid</code> is a <code>xr.Dataset</code> </p><p>I need to do the following:</p>

<pre class="ql-syntax" spellcheck="false">grid['band_data'].plot()
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>